bundles/org.openhab.binding.shelly/README.md

```diff
@@ -0,0 +1,453 @@
+| shelly1          | Shelly Single Relay Switch                             | fully supported                                          |
+| shelly1pm        | Shelly Single Relay Switch with integrated Power Meter | fully supported                                          |
+| shelly1em        | Shelly EM  with integrated Power Meter                 | primarily support                                        |
+| shelly2-relay    | Shelly Double Relay Switch (Shelly2 and Shelly2.5)     | fully supported                                          |
+| shelly2-roller   | Shelly2 in Roller Mode (Shelly2 and Shelly2.5)         | fully supported                                          |
+| shelly2dimmer-white   | Shelly Dimmer in White Mode                       | initial implementation, needs to be verified                                        |
+| shellyht         | Shelly Sensor (temp+humidity)                          | needs to be verified                                     |
+| shellyplug-s     | Shelly Plug                                            | fully supported                                          |
+| shellyplug       | Shelly Plug                                            | fully supported                                          |
+| shellyrgbw2      | Shelly RGB Controller                                  | fully supported                                          |
+| shellybulb       | Shelly Bulb in Color or WHite Mode                     | fully supported                                          |
+| shellysense      | Shelly Motion and IR Controller                        | fully supported                                          |
+| shelly4pro       | Shelly 4x Relay Switch                                 | fully supported                                          |
+| shellysmoke      | Shelly Sensor (temp+humidity)                          | should get discovered, but no special handling yet       |
+
+Feedback is welcome any time. Leave some comments in the forum.
+
+Please send a PM to markus7017 If you encounter errors and include a TRACE log.
+
+
+## Binding installation
+
+### Firmware
+
+To utilize all features the binding requires firmware version 1.5.0 or newer. This should be available for all devices.
+Older versions work in general, but have impacts to functionality (e.g. no events for battery powered devices).
+
+The binding displays a WARNING if the firmware is older. It also informs you when an update is available. Use the device's web ui or the Shelly App to perform the update.
+
+List of Firmware Versions for the different devices could be found here: https://api.shelly.cloud/files/firmware
+
+###m Password protection
+
+For now the binding doesn't support the http authentication. This is on the list. Please deactivate the authentication before discovering devices.
+
+
+### Alpha/Beta versions
```

Comment:
Please remove this block. I understand this is logical when providing it to the users via other ways. But once it's merged this doesn't apply anymore.

Reply:
done, moved all of this to READMEbeta.md